tutorial/dash_deployment_server_examples.py

```diff
@@ -2513,7 +2514,7 @@ def snapshot_page(n_clicks, url, wait_selector):
         you can check your versioning with the command:
         ''')),
 
-        dcc.Markdown('$ pip list', style=styles.code_container),
+        dcc.Markdown('```$ pip list```', style=styles.code_container),
```

Comment:
If you open and close triple backticks on the same line it'll format them as inline code rather than block code. Maybe that's OK, I don't know, but if you want to ensure you get block code formatting, add some newlines:
```suggestion
        dcc.Markdown('```\n$ pip list\n```', style=styles.code_container),
```
(and same in the next bit of diff below)